SourceKitStressTester/Sources/Common/Message.swift

```diff
@@ -492,6 +527,10 @@ extension SourceKitError: CustomStringConvertible {
       return prefix + "<conforming-method-list-offset>" + suffix
     case .collectExpressionType(let document, _):
       return document.modification?.content
+    case .writeModule(let document, _):
+      return document.modification?.content
+    case .interfaceGen(let document,_,  _):
```

Comment:
Nit: the second `_` stole the first's leading whitespace

Reply:
It was greedy :)

Reply:
Speaking of, actually, should I be running swift-format or anything over this?

Reply:
I don't think we ever have for this repo but we probably should at some point (in a separate PR).